flytekit/bin/entrypoint.py

```diff
@@ -495,6 +522,8 @@ def _pass_through():
 @click.option("--dynamic-addl-distro", required=False)
 @click.option("--dynamic-dest-dir", required=False)
 @click.option("--resolver", required=False)
+@click.option("--pickled", is_flag=True, default=False, help="Use this to mark if the distribution is pickled.")
+@click.option("--pkl-file", required=False, help="Location where pickled file can be found.")
```

Comment:
Do we need the `pickled` flag? I think the existance of `pkl-file` is enough to show that the task is pickled.

Reply:
I removed these two arguments from `pyflyte-execute` since I moved the pickle check into the `task_resolver.load_task`.